package.json

```diff
@@ -4,17 +4,22 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "start": "node app.js",
```

Comment:
Why was this added? There is no `app.js` file

Reply:
Deleted